dts/arm/st/stm32f412.dtsi

```diff
@@ -9,6 +9,14 @@
 
 / {
 	soc {
+		usart2: serial@40004400 {
```

Comment:
It was a distraction. I the same file there is definition for node usart3, so I copy-paste to add usart2. Since it is pesent in stm32f4.dtsi it can be removed from dts/arm/st/stm32f412.dtsi